tasks/src/tabular-regression/about.md

```diff
@@ -0,0 +1,17 @@
+## About the Task
+
+Tabular regression is the task of predicting a numerical value given a set of attributes/features. *Tabular* meaning that data is stored in a table (like an excel sheet), and each sample is contained in its own row. The features used to predict our target can be both numerical and categorical. However, including categorical features often requires  additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)). An example of tabular regression would be predicting the weight of a fish given its' species (categorical) and length (numerical).
```

Comment:
I feel like on the last sentence when you write down the type of data at the parenthesis it feels like the emphasis should be on features and not the target. WDYT?

Reply:
Yes I think it is maybe a bit distracting, as tabular regression is defined mostly by the target and not the features.

That being said, do you think the sentences

'''
 The features used to predict our target can be both numerical and categorical. However, including categorical features often requires  additional preprocessing/feature engineering (a few models do accept categorical features directly, like [CatBoost](https://catboost.ai/)).
'''
are unnecessary/distracting?

Reply:
I think I'm fine with the sentence :)